Analysis/HiggsTauTau/src/HTTSequence.cc

```diff
@@ -950,7 +950,7 @@ if(channel != channel::wmnu && !js["store_hltpaths"].asBool()) {
     .set_channel(channel)
     .set_fs(fs.get())
     .set_pair_label("ditau")
-    .set_met_label(met_label)
```

Comment:
Might want to keep it as met_label, with the label being set in the config here: https://github.com/ajgilbert/ICHiggsTauTau/blob/da278ded9041afe130fe8d4418e9661b72a74f4c/Analysis/HiggsTauTau/scripts/configsm2016.json#L47